go.mod

```diff
@@ -138,7 +139,7 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/gnostic v0.5.7-v3refs // indirect
-	github.com/google/go-cmp v0.5.8 // indirect
+	github.com/google/go-cmp v0.5.8
```

Comment:
Shouldn't this move to the first block of require with all the other non-indirect ones?

Reply:
Thanks for informing it. I rerun `go mod tidy` and this package is now in the first required section.